opengever/maintenance/scripts/repository_migration.py

```diff
@@ -859,6 +894,35 @@ def extract_permissions(self, row):
 
         return permissions
 
+    def extract_metadata(self, row):
+        metadata = {}
+        for field in metadata_fields:
+            fieldname = field.getName()
+            value = getattr(row, fieldname)
```

Comment:
`getattr()` without an explicit default as the third argument will raise `AttributeError` (depending on the object - not exactly sure what type `row` is here). Just making sure this is deliberate.

Reply:
row has an attribute for every key in `metadata_fields`.